packages/generator/pyproject.toml

```diff
@@ -49,7 +58,7 @@ warn_unused_configs = true
 warn_unused_ignores = true
 
 [[tool.mypy.overrides]]
-module = "mako.*"
+module = ["mako.*", "tests.utilities.*"]
```

Comment:
Why exclude these?

Reply:
It was added by mistake, while testing. Removing it.

Reply:
ok